### PR TITLE
Cleanup comments and internal variables

### PR DESCRIPTION
**Pull request by dubiousjim**
This is the first of a series of smaller PRs towards #4017.

(I had earlier created #4192, #4193, #4195, #4196, but those were too bloated. Am trying to break them into smaller standalone pieces. However, the later PRs in the sequence are non-trivial to merge with this one. So am pushing this first for review, and once it's merged into master, will base the rest on top of it.)

### CODE REVIEW

File: cli/js/files.ts [hunk 1]
```diff
@@ -215,8 +225,10 @@ export function seekSync(rid: number, offset: number, whence: SeekMode): void {
 
 /** Seek a file ID to the given offset under mode given by `whence`.
  *
- *      const file = await Deno.open("/foo/bar.txt");
- *      await Deno.seek(file.rid, 0, 0);
+ *      (async () => {
+ *        const file = await Deno.open("/foo/bar.txt");
+ *        await Deno.seek(file.rid, 0, 0);
+ *      })();
```
Comment (nayeemrmn) [inline cli/js/files.ts]: This isn't necessary, if you see await you should know async. Especially with TLA support.

Comment (dubiousjim) [reply cli/js/files.ts]: I thought so too. But I was copying over changes to lib.deno.ns.d.t introduced in #4170.

Comment (nayeemrmn) [reply cli/js/files.ts]: Hmm, I can't find a change that adds an async wrapper there.

**Comment by dubiousjim** [comment]
Whoops, you're right. The async() wasn't introduced then. It's been in lib.deno for a while. It was just the process of synchronizing the comments in lib.deno with those in the source files, which was motivated by #4170, that introduced that change to cli/js/files.ts.

I'd be happy to remove the `async() =>` from both places.

File: cli/js/files.ts (hunk 1 above)
Comment (dubiousjim) [reply cli/js/files.ts]: Whoops, you're right. The async() wasn't introduced then. It's been in lib.deno for a while. It was just the process of synchronizing the comments in lib.deno with those in the source files, which was motivated by #4170, that introduced that change to cli/js/files.ts.

I'd be happy to remove the async() => from both places.

**Pull request merged**